lms/djangoapps/api_manager/courses/views.py

```diff
@@ -1539,7 +1539,15 @@ def get(self, request, course_id):  # pylint: disable=W0613
             users_enrolled_qs = users_enrolled_qs.filter(organizations=organization)
             org_ids = [organization]
 
-        users_started = StudentProgress.get_num_users_started(course_key, exclude_users=exclude_users, org_ids=org_ids)
+        group_ids = self.request.QUERY_PARAMS.get('groups', None)
+        if group_ids:
+            group_ids = map(int, group_ids.split(','))
+            users_enrolled_qs = users_enrolled_qs.filter(groups__in=group_ids)
+
+        users_started = StudentProgress.get_num_users_started(course_key,
+                                                              exclude_users=exclude_users,
+                                                              org_ids=org_ids,
+                                                              group_ids=group_ids)
```

Comment:
You need to put `group_ids` filter conditionally like if we have `group_ids` then apply the filter as we are doing in case of users_enrolled above.